topic-operator/src/main/java/io/strimzi/operator/topic/ZkTopicsWatcher.java

```diff
@@ -146,13 +146,17 @@ public void handle(AsyncResult<List<String>> childResult) {
                     tcw.addChild(topicName);
                     tw.addChild(topicName);
                     LogContext logContext = LogContext.zkWatch(TOPICS_ZNODE, watchCount + ":+" + topicName);
-                    topicOperator.onTopicCreated(logContext, new TopicName(topicName)).onComplete(ar -> {
-                        if (ar.succeeded()) {
-                            LOGGER.debug("{}: Success responding to creation of topic {}", logContext, topicName);
-                        } else {
-                            LOGGER.warn("{}: Error responding to creation of topic {}", logContext, topicName, ar.cause());
-                        }
-                    });
+                    if ("__consumer_offsets".equals(created)) {
```

Comment:
I think we should be more general here. AFAIK there is a `__transaction_state` internal topic as well for tracking transaction state. And what about the topics created by Kafka Connect and Kafka streams API? @tombentley wdyt?

Reply:
FTR the issue here is that `__consumer_offsets` and `__transaction_state` are internal in the sense that they can only be created by a broker internally and not directly by any client. Thus Kafka Connect and Kafka Streams "internal" topics are not impacted.